tests/unit/test_classifier_api.py

```diff
@@ -0,0 +1,225 @@
+18: ["On the iterative solution of the gap equation in the Nambu-Jona-Lasinio model In this work we revise the standard iterative procedure to find the solution of the gap equation in the Nambu-Jona-Lasinio model within the most popular regularization schemes available in literature in the super-strong coupling regime. We observe that whereas for the hard cut-off regularization schemes, the procedure smoothly converges to the physically relevant solution, Pauli-Villars and Proper-Time regularization schemes become chaotic in the sense of discrete dynamical systems. We call for the need of an appropriate interpretation of the non-convergence of this procedure to the solution of the gap equation.", 2],
+19: ["Some Remarks on Glaisher-Ramanujan Type Integrals Some integrals of the Glaisher-Ramanujan type are established in a more general form than in previous studies. As an application we prove some Ramanujan-type series identities, as well as a new formula for the Dirichlet beta function at the value $s=3.$", 0]}
+
+TEST_TITLE = 'Pre-images of extreme points of the numerical range, and applications'
+TEST_ABSTRACT = 'We extend the pre-image representation of exposed points of the numerical range of a matrix to all extreme points. With that we characterize extreme points which are multiply generated, having at least two linearly independent pre-images, as the extreme points which are Hausdorff limits of flat boundary portions on numerical ranges of a sequence converging to the given matrix. These studies address the inverse numerical range map and the maximum-entropy inference map which are continuous functions on the numerical range except possibly at certain multiply generated extreme points. This work also allows us to describe closures of subsets of 3-by-3 matrices having the same shape of the numerical range.'
+
+
+
+def make_test_dataset():
+    test_dataframe = pd.DataFrame.from_dict(TRAINING_DATA, orient='index', columns=['texts', 'coreness'])
+    test_dataframe.to_pickle(os.path.join(current_app.config['CLASSIFIER_DATAFRAME_PATH']))
+
+
+def download_pretrained_wikitext_language_model_and_itos():
```

Comment:
you shouldn't have to do this in the test if the api already does it for you. In general, copy/paste code from the application to the tests shouldn't happen.